services/svc/blobstorage/src/package_2021_02/mod.rs

```diff
@@ -254,7 +254,14 @@ pub mod service {
     }
     pub mod get_properties {
         use super::models;
-        type Response = models::StorageServiceProperties;
+        struct Response(azure_core::Response);
+        impl Response {
+            pub async fn into_body(self) -> azure_core::Result<models::StorageServiceProperties> {
```

Comment:
It is now an additional `into_body` to get the response body. If typed headers are available, they will be made available via `headers(&self)` in a future pull request.